src/Index.js

```diff
@@ -923,6 +923,69 @@ Index.prototype.exists = function(callback) {
   });
 };
 
+Index.prototype.findObject = function(findCallback, requestOptions, callback) {
+  requestOptions = requestOptions === undefined ? {} : requestOptions;
+  var paginate = requestOptions.paginate !== undefined ? requestOptions.paginate : true;
```

Comment:
would suggest something like "allPages"

Reply:
It's on the API Client specs: https://github.com/algolia/algoliasearch-client-specs-internal/blob/master/api-client-specifications/search_api.md#search_index-interface

```
To prevent the iteration through pages of results, `paginate` in
request_options can be set to false. This will stop the function at the end of
the first page of search results even if no object does match.
```